src/pages/area-management/area-management.jsx

```diff
@@ -65,9 +68,31 @@ const AreaManagement = () => {
   };
 
   useEffect(() => {
-    TLPBackend.get('/areas/area-management').then(res => {
-      setAreaResponseData(res.data);
-    });
+    async function fetchAreas() {
+      await TLPBackend.get('/areas')
+        .then(res => {
+          setTimeout(() => {
+            addAssociatedSiteToArea(res.data);
```

Comment:
🚫 **[eslint]** <[no-undef](https://eslint.org/docs/rules/no-undef)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'addAssociatedSiteToArea' is not defined.